File: DAY-5/Q1.c
#include<stdio.h>
int main(){
    int n,sum=0;
    printf("Enter number to checked perfect number:");
    scanf("%d",&n);
    for(int i=1;i<n;i++){
        if(n%i==0){
            sum=sum+i;
        }
    }
   if(sum==n){
    printf("%d is perfect Number",n);
   }
   else{
    printf("Not a perfect Number");
   }
 return 0;
}

File: DAY-16/Q1.java
import java.util.Scanner;

public class Q1{
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter Size of array: ");
        int size = sc.nextInt();

        int[] arr = new int[size];
        int sum = 0;

        System.out.println("Enter Array Elements: ");
        for(int i=0;i<size;i++){
            arr[i] = sc.nextInt();
            sum+= arr[i];
        }

        int n = size + 1;

        int TotalSum = n*(n+1)/2;
        
        int missing = TotalSum - sum ;
        System.out.println("Missing Number = " +missing);

    }
}   